frameworks/g-cloud-9/manifests/edit_submission.yml

```diff
@@ -237,14 +232,14 @@
     - multiqStandardsCSASTAR
     - creditOrDebitCardInformation
     - multiqStandardsPCI
-    - standardsOther
```

Comment:
Is there a `standardsOther` question file that needs deleting too, or is it used elsewhere?